### PR TITLE
Milestone 3: Add Contact form

### PR DESCRIPTION
**Pull request by TumiRantsane**
## Add the contact form and finish the Mobile version.
- Make sure that:
  - Name field accepts up to 30 chars.
  - Email field accepts only emails.
  - Message field accepts up to 500 chars.
  - All fields have placeholder text.
  - All fields have the correct type.
  - A user cannot submit the form if one or more of the fields is empty.
- To collect user data, use the [Formspree service](https://formspree.io/html).
- To lay out the elements, use Flexbox.
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,186 +2,251 @@
...
+
+      <form class="contact-form" action="https://formspree.io/f/mrgdrqjv" method="POST">
+        <ul>
+          <li class="user-input">
+            <label for="name">Your name</label>
+            <input type="text" name="user-name" id="name" placeholder="John Doe" maxlength="30" required>
+          </li>
+          <li class="user-input">
+            <label for="email">Your email</label>
+            <input type="text" name="user-email" id="email" placeholder="example@gmail.com" required>
```
Comment (Qoosim) [inline index.html]: - [ ] The `<input type="email">` defines a field for an e-mail address. The input value is automatically validated to ensure it is a properly formatted e-mail address. Kindly attend to that and fix it.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Descriptive PR title and summary. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MNisarAli) [reply index.html]: Thanks for your review. I have made the required changes to `<input type=>` as `<"email">`

**Review by Debas-31 — APPROVED** [review]
> ## Hi @MNisarAli  :wave:, 

# Status: Approved :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Project functionality :heavy_check_mark: 
- UI design :heavy_check_mark: 
- Figma template usage :heavy_check_mark: 

## Optional suggestions
- [ ] No

***Cheers and Happy coding!👏👏👏***

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -198,6 +197,56 @@ <h3 class="skill-cat">Skills</h3>
       </ul>
     </section>
   </main>
+  <footer>
+    <section class="contact-section">
+      <h2 class="get-started">Get started</h2>
```
Comment (DianaBeki) [inline index.html:202]: - Please i suggest in your h2 element you would have used a class-like heading-2 for someone to understand your code easily while styling.

**Pull request merged**